src/benchmark/managers/config.py

```diff
@@ -85,6 +86,9 @@ def get_execution_options(
             # It means we are not yet ready. Return None
             # This check also serves to ensure we have only one valid relation at the time
             return None
+
+        peer_list = self.peers
+        peer_list.sort()
```

Comment:
otherwise we get different configuraiton every time, and we use the config output to decide which stage are we.